src/components/IndiaStateMap/Switch.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+const Switch = ({ mode, setMode, activeAttribute }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="switch">
+        <button
+          className={
+            "btn btn-chart " + (mode === 'normal' ? activeAttribute : "")
+          }
+          onClick={() => { setMode('normal') }}
+        >
+          Normal
+      </button>
+        <button
+          className={
+            "btn btn-chart " + (mode === 'per million' ? activeAttribute : "")
```

Comment:
Let's remove the space between `per million`.

```suggestion
            "btn btn-chart " + (mode === 'per-million' ? activeAttribute : "")
```

Reply:
Done